app/controllers/AnnotationIOController.scala

```diff
@@ -135,6 +131,55 @@ Expects:
       }
   }
 
+  private def mergeAndSaveVolumeLayers(volumeLayersGrouped: Seq[List[UploadedVolumeLayer]],
+                                       client: WKRemoteTracingStoreClient,
+                                       otherFiles: Map[String, TemporaryFile]): Fox[List[AnnotationLayer]] = {
+    if (volumeLayersGrouped.isEmpty) return Fox.successful(List())
+    if (volumeLayersGrouped.length > 1 && volumeLayersGrouped.exists(_.length > 1))
+      return Fox.failure("Cannot merge multiple annotations that each have multiple volume layers.")
+    if (volumeLayersGrouped.length == 1) { // Just one annotation was uploaded, keep its layers separate
```

Comment:
I'm not sure if I'm understanding `Just one annotation was uploaded, keep its layers separate`. There can still be multiple annotations, but each of those has a different layer, right?